lib/manager/dockerfile/extract.spec.ts

```diff
@@ -61,15 +61,24 @@ describe(getName(), () => {
         'FROM registry2.something.info:5005/node:8\n'
       ).deps;
       expect(res).toMatchSnapshot();
-      expect(res[0].depName).toEqual('registry2.something.info:5005/node');
+      expect(res[0].depName).toEqual('node');
```

Comment:
Should we validate registry urls here too?

Reply:
Maybe, although they'll be caught later anyway